packages/amplify_datastore/CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.2.3 (2021-09-09)
+
+### Features
+
+- feat(auth): OIDC/Lambda Support (#777)
+- feat(datastore): Add start and stop APIs (#811)
+- feat(auth): add options to resendSignUpCode (#738)
```

Comment:
oops might've mis-included changes logs from auth category?

Reply:
I think this one we placed within the scope of 'auth' but actually impacted many packages.

Reply:
This PR included a version bump of amplify-android, so it did impact categories outside of auth. I'm not sure how we want to handle that in the future.